src/client/components/MainFooter/MainFooter.tsx

```diff
@@ -24,71 +24,72 @@ interface FooterBlockProps {
 }
 
 function FooterBlock({ id, title, links, description }: FooterBlockProps) {
-  const titleRole = useDesktopScreen() ? 'columnheader' : 'button';
   return (
-    <Grid.Cell span={4} className={'page-footer-space-y '}>
-      <Heading
-        inverseColor={true}
-        level={4}
-        role={titleRole}
-        className="ams-mb--xs"
-      >
+    <Grid.Cell key={title} span={4} className="page-footer-space-y">
+      <Heading inverseColor level={4} className="ams-mb--xs">
         {title}
       </Heading>
       {!!description && getEl(description as AstNode)}
       {!!links.length && (
-        <UnorderedList markers={false}>
+        <LinkList>
           {links.map((link) => (
-            <LinkList key={link.title}>
-              <LinkList.Link
-                variant="inList"
-                onBackground={'dark'}
-                href={link.to}
-              >
-                {link.title}
-              </LinkList.Link>
-            </LinkList>
+            <LinkList.Link
+              key={link.to}
+              variant="inList"
+              onBackground="dark"
+              href={link.to}
+            >
+              {link.title}
+            </LinkList.Link>
           ))}
-        </UnorderedList>
+        </LinkList>
       )}
     </Grid.Cell>
   );
 }
-function getEl(astElement: AstNode | AstNode[], index: number = 0) {
-  if (Array.isArray(astElement) && astElement.length > 0) {
+
+function getEl(astElement: AstNode | AstNode[], i: number = 0) {
+  let index = i;
+  if (Array.isArray(astElement)) {
     return astElement.map((el, index) => getEl(el, index));
   }
 
   if ('type' in astElement && astElement.type === 'text') {
-    return <>{astElement.content || ''}</>;
+    return <span>{astElement.content || ''}</span>;
   }
 
   if ('type' in astElement && astElement.type === 'tag') {
-    const children = astElement.children ? getEl(astElement.children) : null;
+    const children = astElement.children
+      ? getEl(astElement.children, index)
+      : null;
 
     switch (astElement.name) {
       case 'a':
         return (
-          <MaLink
+          <Link
+            onBackground="dark"
+            variant="standalone"
             key={index}
```

Comment:
Ik zou de href als index gebruiken, https://medium.com/geekculture/reactjs-why-index-as-a-key-is-an-anti-pattern-4b9dc6ef0067